docs/iris/src/developers_guide/documenting/docstrings.rst

```diff
@@ -7,7 +9,7 @@ appropriate docstring.
 
 This document has been influenced by the following PEP's,
 
-   * Attribute Docstrings `PEP-224 <http://www.python.org/dev/peps/pep-0224/>`_ 
+   * Attribute Docstrings `PEP-224 <http://www.python.org/dev/peps/pep-0224/>`_
```

Comment:
@trexfeathers You could use this `sphinx` inline directive instead:
```
:pep:`224`
```
Reference [here](https://www.sphinx-doc.org/en/1.7/markup/inline.html#other-semantic-markup)

Reply:
88fb0710564dc4b69496cbd5da5e76d40ae21024